gulpconfig.js

```diff
@@ -77,68 +77,6 @@ module.exports = {
       'jpeg',
       'png',
     ],
-    twigNamespaces: {
```

Comment:
this file isn't currently used at all anymore, but i'm taking bits of config from it as needed into the local gulp task files